react.js

```diff
@@ -35,6 +56,11 @@ this.state = {
   count: 0
 }
 
+handleClick = () => {
+    this.setState(({ count }) => ({
+      count: count + 1
+    }));
+
 
```

Comment:
1) class Recipes extends Component
2) recipes should be an array so we can use map
3) destructuring recipes out of state
4) the map logic should be in the render but not the return